src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.html

```diff
@@ -0,0 +1,42 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container class="modal-title"
+                i18n>Raise an Issue</ng-container>
+
+  <ng-container class="modal-content">
+    <form class="feedbackForm" [formGroup]="feedbackForm" (ngSubmit)="onSubmit()">
+      <div class="formComp">
+        <label for="project">Project: </label>
+        <select formControlName="project" class="form-control">
+          <option disabled>Select Project</option>
+          <option>Choose Project</option>
+          <option *ngFor="let projectName of project">{{projectName}}</option>
+        </select>
+        <div *ngIf="f.project.touched && f.project.invalid" class="alert alert-danger">
+          <div *ngIf="f.project.errors.required">Project is required.</div>
+        </div>
+      </div>
+      <div class="formComp">
+        <label for="tracker">Tracker: </label>
+        <select formControlName="tracker" class="form-control">
+          <option disabled>Select Tracker</option>
+          <option>Choose Tracker</option>
+          <option *ngFor="let trackerName of tracker">{{trackerName}}</option>
+        </select>
+        <div *ngIf="f.tracker.touched && f.tracker.invalid" class="alert alert-danger">
+          <div *ngIf="f.tracker.errors.required">Tracker is required.</div>
+        </div>
+      </div>
+      <div class="formComp">
+        <label for="subject">Subject: </label>
+        <input id="subject" type="text" formControlName="subject" class="form-control" placeholder="Add issue Title">
+      </div>
+      <div class="formComp">
+        <label for="description">Description: </label>
+        <input id="description" type="text" formControlName="description" class="form-control" placeholder="Add issue description">
```

Comment:
Since the description is a bigger field, wouldn't it make more sense to make this a textarea?
See: https://getbootstrap.com/docs/4.0/components/forms/
```suggestion
      <textarea id="description" type="text" formControlName="description" class="form-control" placeholder="Add issue description">